index.html

```diff
@@ -0,0 +1,537 @@
+                      Benkler studies commors-based peer production, and published his seminal book The wealth of networks in 2006.
+                    </p>
+                  </div>
+                </div>
+              </div>
+
+              <div class="col-sm-6 guests-container">
+                <div class="row d-flex">
+                  <div class="col-4 position-relative">
+                    <img src="img/speaker2.png" alt="Profile" />
+                    <img
+                      src="img/bg-no-color.png"
+                      class="bg-profile"
+                      alt="Profile"
+                    />
+                  </div>
+                  <div class="col-8">
+                    <h5 class="h4 font-weight-bold">Killnam Chon</h5>
+                    <p
+                      class="
+                        speakers
+                        position-relative
+                        pb-2
+                        c-primary
+                        font-italic font-weight-bold
+                      "
+                    >
+                      <span></span>
+                    </p>
+                    <p>
+                      Killnam Chon helped bring the internet to asia and is an outspoken advocate for the open web and digital commons. In 2012, he was included into the inaqual class of the open society (ISOO) Internet hall of fame.
+                    </p>
+                  </div>
+                </div>
+              </div>
+
+              <div class="col-sm-6 guests-container">
+                <div class="row d-flex">
+                  <div class="col-4 position-relative">
+                    <img src="img/speaker3.png" alt="Profile" />
+                    <img
+                      src="img/bg-no-color.png"
+                      class="bg-profile"
+                      alt="Profile"
+                    />
+                  </div>
+                  <div class="col-8">
+                    <h5 class="h4 font-weight-bold">sohYeong noh</h5>
+                    <p
+                      class="
+                        speakers
+                        position-relative
+                        pb-2
+                        c-primary
+                        font-italic font-weight-bold
+                      "
+                    >
+                      <span>Director of art center Nairobi nd a board member of CC Korea</span>
+                    </p>
+                    <p>
+                      As the main venue for new medea art production in Korea, Nabi promotes cross-disciplinairy collaboration and understanding among science technology, humanitie and the arts.
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="col-sm-6 guests-container">
+                <div class="row d-flex">
+                  <div class="col-4 position-relative">
+                    <img src="img/speaker4.PNG" alt="Profile" />
```

Comment:
- [ ] The image link is breaking, please kindly fix that, see image below.
![kingJulia](https://user-images.githubusercontent.com/65852621/138378723-9b1eea83-8026-43af-8e6e-0fbd33ae7038.png)